test/functional/host_test.rb

```diff
@@ -87,6 +87,19 @@
     assert_cmd(expected_result, result)
   end
 
+  it "accepts host parameter attributes" do
+    params = ['--parameters', 'param1=value1']
+
+    api_expects(:hosts, :create, 'Create host with parameters') do |par|
+      par['host']['host_parameters_attributes'][0]['name'] == 'param1'
+      par['host']['host_parameters_attributes'][0]['value'] == 'value1'
```

Comment:
BTW this code tests only the last line, you need to add `&&` at the end of the previous line.